src/common/tests/btree.sh

```diff
@@ -72,14 +72,27 @@ done
 set -x
 set -e
 
+gen_test_conf_string()
+{
+        name=""
+        [ ! -z "$1" ] && name="${name} IDIR=$1"
+        [ ! -z "$2" ] && name="${name} IPL=$2"
+        [ ! -z "$3" ] && name="${name} PMEM=$3"
+        echo $name
+}
+
 run_test()
 {
     printf "\nOptions: IPL='%s' IDIR='%s' PMEM='%s'\n" "$IPL" "$IDIR" "$PMEM"
     if [ -z ${PERF} ]; then
 
+        test_conf=$(gen_test_conf_string ${IDIR} ${IPL} ${PMEM})
```

Comment:
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Double quote to prevent globbing and word splitting. [SC2086]